onnxruntime/python/tools/quantization/qdq_loss_debug.py

```diff
@@ -222,3 +230,91 @@
             act_values["float"] = float_acts
 
     return qdq_cmp
+
+
+_QUANT_WEIGHT_SUFFIX = "_quantized"
```

Comment:
could you please make it as a constant in the quant_utils and replace all the "_quantized" with the constant variable